source/Octopus.Client/Octopus.Client.MacOs.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
As above, this is not mac specific - perhaps call it "Local" or "Unsigned" instead of "MacOs.nuspec"